airbyte-integrations/connectors/source-typeform/source_typeform/source.py

```diff
@@ -204,20 +204,20 @@ def check_connection(self, logger: AirbyteLogger, config: Mapping[str, Any]) ->
             form_ids = config.get("form_ids", []).copy()
             # verify if form inputted by user is valid
             try:
-                url = f"{TypeformStream.url_base}/me"
+                url = f"{TypeformStream.url_base}me"
```

Comment:
maybe use `urljoin` function to ensure this won't break in any case

Reply:
done